src/plugins/intel_cpu/src/node.cpp

```diff
@@ -1302,16 +1301,11 @@ Node* Node::NodesFactory::create(const std::shared_ptr<ngraph::Node>& op, const
     };
     Node *newNode = nullptr;
     std::string errorMessage;
-    {
-        std::unique_ptr<Node> ol(createNodeIfRegistered(intel_cpu, Type::Generic, op, context));
-        if (ol != nullptr && ol->created(context->getExtensionManager()))
-            newNode = ol.release();
-    }
 
     if (newNode == nullptr) {
         try {
             std::unique_ptr<Node> ol(createNodeIfRegistered(intel_cpu, TypeFromName(op->get_type_name()), op, context));
-            if (ol != nullptr && ol->created(context->getExtensionManager()))
+            if (ol != nullptr)
```

Comment:
```suggestion
            if (ol != nullptr && ol->created())
```
We still need to check "ol->created()".